apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1101,9 +1174,81 @@ static void tc_fs_vfs_statfs(void)
 	ret = statfs(NULL, &fs);
 	TC_ASSERT_EQ("statfs", ret, ERROR);
 
+	/*root pseudo file system */
+	ret = statfs("/dev", &fs);
+	TC_ASSERT_EQ("statfs", ret, OK);
+
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+* @testcase         tc_fs_vfs_fstat
+* @brief            Get status of specific file
+* @scenario         Get status of specific file(VFS_FILE_PATH) by stat
+* @apicovered       fstat
+* @precondition     File VFS_FILE_PATH should be existed
+* @postcondition    NA
+*/
+static void tc_fs_vfs_fstat(void)
+{
+	char *filename = VFS_FILE_PATH;
+	struct stat st;
+	int ret;
+	int fd;
+	fd = open(filename, O_RDWR);
+	TC_ASSERT_GEQ("open", fd, 0);
+
+	ret = fstat(fd, &st);
+	TC_ASSERT_EQ("fstat", ret, OK);
```

Comment:
@kavyasrinunna
 Please check if "st" also needs to be verified

Reply:
I think "st" may not be  null, so verifying "st" may not be required.